testdata/scripts/reverse.txt

```diff
@@ -121,19 +131,23 @@ var _ = reflect.TypeOf(UnobfuscatedStruct{})
 
 var _ = struct{SomeField int}(UnobfuscatedStruct{})
 -- reverse.stdout --
+lib filename: test/main/lib/lib.go
+
 goroutine 1 [running]:
 runtime/debug.Stack(0x??, 0x??, 0x??)
 	runtime/debug/stack.go:24 +0x??
 test/main/lib.printStackTrace(0x??, 0x??, 0x??, 0x??)
-	test/main/lib/lib.go:23 +0x??
-test/main/lib.(*ExportedLibType).ExportedLibMethod(...)
-	test/main/lib/lib.go:12
-main.unexportedMainFunc.func1()
-	test/main/main.go:16 +0x??
+	test/main/lib/lib.go:28 +0x??
+test/main/lib.(*ExportedLibType).ExportedLibMethod(0x??, 0x??, 0x??, 0x??, 0x??)
```

Comment:
I have no idea why this particular line changed. I guess the func is larger so its stack is different?